### PR TITLE
update xda developers links + a7 2017 support

### PR DESCRIPTION
**Pull request by SirRGB**
* migrate xda developers link to a new url
* add A7 2017 support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,12 +57,12 @@ Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 L
...
         - The recommended way to install MicroG is to use the zip file provided here: [https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
       - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
-    - or you can just search the web or the [xda-developers forum](https://forum.xda-developers.com) for an appropriate version for your device.
+    - or you can just search the web or the [xda-developers forum](https://xdaforums.com/) for an appropriate version for your device.
 3. Start the desktop app and follow the instructions.
 
 ## Officially supported devices
 
-Currently, the **we support 73 devices** by various vendors and working on adding more soon!
+Currently, the **we support 74 devices** by various vendors and working on adding more soon!
```
Comment (tsterbak) [inline README.md:65]: Did you recount it? Maybe I made an error there, then I should also update the website.

Comment (SirRGB) [reply README.md:65]: nope, added a7 2017 support, so I thought Id just increase it by one
It is essentially the same as the a5 2017, so I just pasted the configs and called it a day

File: README.md [hunk 2]
```diff
@@ -57,12 +57,12 @@ Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 L
       - [MicroG](https://microg.org)
         - The recommended way to install MicroG is to use the zip file provided here: [https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
       - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
-    - or you can just search the web or the [xda-developers forum](https://forum.xda-developers.com) for an appropriate version for your device.
+    - or you can just search the web or the [xda-developers forum](https://xdaforums.com/) for an appropriate version for your device.
```
Comment (MagicLike) [inline README.md:60]: Just a cosmetic thing: Would you mind removing the trailing slashes from all the XDA Links?

File: openandroidinstaller/assets/configs/a7y17lte.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+metadata:
+  maintainer: Tobias Sterbak (tsterbak)
```
Comment (MagicLike) [inline openandroidinstaller/assets/configs/a7y17lte.yaml:2]: Oh and you should also put your name / nickname here, as you are the maintainer and creator of the config :D

**Review by MagicLike — CHANGES_REQUESTED** [review]

**Review by MagicLike — APPROVED** [review]
LGTM! :D

**Pull request merged**